### ISSUE TITLE
Consider removing the "btn-social" class for custom OAuth2 providers (and potentially others)

### ISSUE THREAD

**Issue by ldruschk** [hackmdio/codimd#1491]
I will preface this issue by saying that this problem is caused by using an ad-blocker and thus might be out of scope for CodiMD (and therefore I am not creating a pull request directly).

The "btn-social" class is blocked by the "Fanboy's Social" and "Fanboy's Annoyance" filter lists which are available in e.g. uBlock origin by default (although I am not sure if they are enabled by default). Therefore the "Sign in via ..." button is not visible.

While this is most likely intended behavior for e.g. the Facebook, Twitter and Google buttons (and potentially some others) and people who enable these filter lists probably won't miss them, this also blocks self-hosted solutions like custom OAuth2 providers.

It should however be noted that this changes the style of the button (the button on the top in the picture is with the `btn-social` class, the bottom one without the class).

![Screenshot from 2020-04-20 18-40-31](https://user-images.githubusercontent.com/14059613/79776816-946f0d80-8336-11ea-9499-b60d80db5570.png)
